dev/bots/firebase_testlab.sh

```diff
@@ -8,11 +8,9 @@
 # that the app doesn't crash upon startup.
```

Comment:
it might be a good idea to move this to test.dart rather than having its own separate shell script.

Reply:
Indeed that would be better.  I split the difference and dumbed down .cirrus.yml and pushed the subshard -> test logic into test.dart.  firebase_testlab.sh still lives, though.